subt/script/check_rosK2.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Maybe it should not be called *.sh when it is not actually sh-script but bash-script?

Reply:
That makes sense. Maybe it could be a future impulse to rename the others bash scripts with *.sh endswith.